notebooks/regression/tutorial_regression.md

```diff
@@ -399,6 +401,8 @@ the Multilayer Perceptron has two hidden dense layers with 20 neurons each follo
 
 
 ```python
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3' 
```

Comment:
Maybe a comment on this ? Can we move this line of code after all imports ?

Reply:
It allows you to disable all debugging logs from Tensorflow and must be placed *before* importing the Tensorflow dependencies...